demo/app/lib/assets/walletConnectIssuerConfig.json

```diff
@@ -1,8 +1,9 @@
 {
   "TrustBloc Staging Issuer" : {
```

Comment:
I would suggest renaming this file to make it clear that this is for the wallet-initiated flow. Something like `walletInitiatedIssuerConfig`